### PR TITLE
Add a question related to pre submission enquiries in the issue template 

### PR DESCRIPTION
**Pull request by maelle**
So that we might never forget those when the package eventually gets submitted.

@ropensci/editors

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -21,6 +21,8 @@
 -   Are there other R packages that accomplish the same thing? If so, how does
 yours differ or meet [our criteria for best-in-category](https://github.com/ropensci/onboarding/blob/master/policies.md#overlap)?
 
+-   If you had made a successful pre-submission enquiry, please copy below the link to the corresponding issue.
```
Comment (noamross) [inline issue_template.md]: How about: If you had made a successful pre-submission enquiry, please paste the link to the corresponding issue _(or forum post, twitter thread, or other discussion)_.

Comment (maelle) [reply issue_template.md]: Great point. Should we also suggest to tag the editor in case of a private discussion 🤔

Comment (noamross) [reply issue_template.md]: Sure, let's see how to do this with minimum wordiness:

>If you made a pre-submission enquiry, please paste the link to the corresponding issue, forum post, or other discussion, or `@tag` the editor you contacted.

Comment (maelle) [reply issue_template.md]: perfect! 😸

**Pull request merged**